src/DynamoInstall/Product.wxs

```diff
@@ -78,12 +78,12 @@
     <DirectoryRef Id="ApplicationProgramsFolder">
       <Component Id="ApplicationShortcut" Guid="d7575401-f8ba-488f-b5e4-5ad24dbc4b10" Win64="yes">
         <Shortcut Id="ApplicationStartMenuShortcut"
-                  Name="Dynamo 0.7"
+                  Name="Dynamo 0.8"
                   Description="Dynamo Sandbox"
                   Target="[DYNAMO_INSTALLDIR]\DynamoSandbox.exe"
                   WorkingDirectory="DYNAMO_INSTALLDIR"/>
         <RemoveFolder Id="ApplicationProgramsFolder" On="uninstall"/>
-        <RegistryValue Root="HKCU" Key="Software\Microsoft\Dynamo_07" Name="installed" Type="integer" Value="1" KeyPath="yes"/>
+        <RegistryValue Root="HKCU" Key="Software\Microsoft\Dynamo_08" Name="installed" Type="integer" Value="1" KeyPath="yes"/>
```

Comment:
@sharadkjaiswal if it's not there the WIX proj can't compile because the shortcut component require a HKCU key.
it's not used anywhere at all